app/controllers/students_controller.rb

```diff
@@ -0,0 +1,16 @@
+class StudentsController < ApplicationController
+  def index
+    students = Student.all
+    render json: students
+  end
+
+  def grades
+    students_by_grade = Student.all.order(grade: :desc)
```

Comment:
You can omit the `.all` here 
```suggestion
    students_by_grade = Student.order(grade: :desc)
```